documentation.html.haml

```diff
@@ -4,21 +4,73 @@ rightcol_partial: rightcol-doc-versions.html.haml
 title: Documentation
 ---
 %h2 Infinispan community documentation
-%p Welcome to the Infinispan documentation, straight from the source! Here you can find help and knowledge provided by the community of contributors.
+%p Welcome to the Infinispan documentation, straight from the source!
 
-%p To find available documentation for Infinispan, visit the index page for your version.
+%a.btn.btn-primary.btn-block{:href =>site.ispn.stable.docs.start} Getting Started with Infinispan
 
-%i.icon-circle-arrow-right
-%a{:href =>"/release-notes"} Review the Release Notes here.
+%br/
 
-%h3 Join us
+.row
+  .col-md-4
+    .panel.panel-default{:style => "height:225px;width:275px"}
+      .panel-heading
+        %i.icon-terminal
+        Admin
+      .panel-body
+        %ul.list-unstyled
+          %li
+            %a{:href=>site.ispn.stable.docs.server} Infinispan Server Guide
+          %li
+            %a{:href=>site.ispn.stable.docs.cli} Command Line Interface
+          %li
+            %a{:href=>site.ispn.stable.docs.upgrade} Upgrading Infinispan
+          %li
+            %a{:href=>site.ispn.stable.docs.security} Security
+          %li
+            %a{:href=>site.ispn.stable.docs.tuning} Performance Tuning
+
+  .col-md-4
+    .panel.panel-default{:style => "height:225px;width:275px"}
+      .panel-heading
+        %i.icon-code
+        Developers
+      .panel-body
+        %ul.list-unstyled
+          %li
+            %a{:href=>site.ispn.stable.docs.embedded} Embedding Infinispan
+          %li
+            %a{:href=>site.ispn.stable.docs.dev} Developer's Guide
+          %li
+            %a{:href=>site.ispn.stable.docs.config} Configuring Infinispan
+          %li
+            %a{:href=>site.ispn.stable.docs.xsite} Cross-Site Replication
+          %li
+            %a{:href=>site.ispn.stable.docs.rest} REST API
+          %li
+            %a{:href=>site.ispn.stable.docs.hotrod} Hot Rod Java Client Guide
+          %li
+            %a{:href=>site.ispn.stable.docs.integrations} Integrations
+
+  .col-md-4
+    .panel.panel-default{:style => "height:225px;width:275px;"}
+      .panel-heading
+        %i.icon-cloud
+        Cloud
+      .panel-body
+        %ul.list-unstyled
+          %li
+            %a{:href=>site.ispn.stable.docs.operator} Infinispan Operator Guide
+          %li
+            %a{:href=>site.ispn.stable.docs.image} Infinispan Server Image
+
+%br/
+%h4 Join us
 %p Infinispan is actively developed and maintained by a community of generally cool people. We welcome you to chat with us, report issues, suggest changes, or ask questions.
 %i.icon-comments-alt
 %a(href="/community") Join the Infinispan community!
-
-%h3 Contribute to Infinispan
-%p Got an improvement to the source code or want to write some docs?
+%br/
+%br/
 %i.icon-github
-%a{:href => site.ispn.unstable.docs.contributing} Review the Contributor's Guide and submit a PR.
+%a{:href => site.ispn.stable.docs.contributing} See our Contributor's Guide and submit a PR.
```

Comment:
I think it makes more sense to link the contribution guide from the unstable release

Reply:
gotcha. I fixed one other link for the contributors guide so I think they all point to unstable now.